Source/WebCore/PAL/pal/text/TextCodecUTF8.cpp

```diff
@@ -419,16 +431,17 @@ String TextCodecUTF8::decode(std::span<const uint8_t> bytes, bool flush, bool st
                         auto chunk = *reinterpret_cast_ptr<const WTF::MachineWord*>(source.data());
                         if (!WTF::containsOnlyASCII<LChar>(chunk))
                             break;
-                        copyASCIIMachineWord(destination16, source.data());
+                        copyASCIIMachineWord(destination16, source);
                         source = source.subspan(sizeof(WTF::MachineWord));
-                        destination16 += sizeof(WTF::MachineWord);
+                        destination16 = destination16.subspan(sizeof(WTF::MachineWord));
                     }
                     if (source.empty())
                         break;
                     if (!isASCII(source.front()))
                         continue;
                 }
-                *destination16++ = source.front();
+                destination16[0] = source.front();
```

Comment:
How do you decide when to use `front()` and when to use `[0]`?

Reply:
They are equivalent as far as I know. Maybe I should just stick to [0] since it is more concise?